pkg/types/ibmcloud/validation/platform_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
+	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/installer/pkg/types/ibmcloud"
```

Comment:
```suggestion
	"k8s.io/apimachinery/pkg/util/validation/field"
	configv1 "github.com/openshift/api/config/v1"
	
	"github.com/openshift/installer/pkg/types/ibmcloud"
```

Reply:
`gofmt` preferred this as is, so I've changed it back
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_installer/7632/pull-ci-openshift-installer-master-gofmt/1721621569781895168